build.sbt

```diff
@@ -20,6 +20,11 @@ lazy val databaseMigration = (project in file("database/migration"))
   .dependsOn(core)
   .withTestSettings
 
+lazy val dockerHashing = (project in file("dockerHashing"))
+  .settings(dockerHashingSettings: _*)
+  .dependsOn(core)
```

Comment:
I'm realizing that this is going to make the `updateDocker` PR much harder to rebase as it was trying to abstract most of the boilerplate code for handling enqueuing failures etc.. in a trait reused by both the docker actor and the I/O actor...
Anyway, If it turns out to be a giant pain to cut the dependency I would not bother too much..